src/overlays/actors/ovl_En_Bx/z_en_bx.c

```diff
@@ -21,13 +22,215 @@ const ActorInit En_Bx_InitVars = {
     (ActorFunc)EnBx_Update,
     (ActorFunc)EnBx_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Bx/EnBx_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Bx/EnBx_Destroy.s")
+static ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_UNK6, 0x11, 0x09, 0x00, 0x00, COLSHAPE_CYLINDER },
+    { 0x01, { 0xFFCFFFFF, 0x03, 0x04 }, { 0xFFCFFFFF, 0x01, 0x00 }, 0x01, 0x01, 0x00 },
+    { 60, 100, 100, { 0, 0, 0 } },
+};
+
+static ColliderQuadInit sQuadInit = {
+    { COLTYPE_UNK10, 0x11, 0x00, 0x00, 0x00, COLSHAPE_QUAD },
+    { 0x00, { 0xFFCFFFFF, 0x03, 0x04 }, { 0x00000000, 0x00, 0x00 }, 0x01, 0x00, 0x00 },
+    { { { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f } } },
+};
+
+Vec3f D_809D250C = { 0.0149999996647f, 0.0149999996647f, 0.0149999996647f };
+
```

Comment:
as ethan said in discord these floats can probably be cleaned up. the precision the converter has is not the same as what the compiler uses. 0.015 probably works for all 3